test/unit/tools/eslint.rb

```diff
@@ -0,0 +1,25 @@
+module Test
+  module Quality
+    module Tools
+      # Test for the 'bigfiles' tool within the quality gem
+      module Eslint
+        def eslint_expected_args
+          'fake1.js src/foo/testfake1.js features/featuresfake1.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.